packages/ember-views/lib/views/view.js

```diff
@@ -994,6 +995,29 @@ var View = CoreView.extend(
   },
 
   /**
+    @private
+
+    Renders the view into `body` element and returns the element.
+
+    This method is useful if you want to render the view into an element that
+    is not in the document's body. Instead, a new `body` element, detached from
+    the DOM is returned. FastBoot uses this to serialize the rendered view into
+    a string for transmission over the network.
+
+    @method renderToElement
+    @return {HTMLBodyElement} element
+  */
+  renderToElement: function() {
+    var element = this.renderer._dom.createElement('body');
+
+    run(this, function() {
+      this.renderer.appendTo(this, element);
+    });
```

Comment:
@mixonic My goal here was to ensure that the entire rendering process had completed before returning the element, since in FastBoot™ we immediately serialize the returned element and send it down the socket. We could, of course, push the run loop explicitness out to the consumer of the API—it just felt like every use of `renderToElement` would probably want the same semantics, and making the user remember to wrap it in a run loop seemed annoying.